src/Reader/Reader.php

```diff
@@ -695,7 +699,7 @@ protected static function registerCoreExtensions()
      * @param  array
      * @return array
      */
-    public static function arrayUnique(array $array)
+    public static function arrayUnique(array $array): array
```

Comment:
Please do not add php typehints as this will is a BC break unless the class is final.